src/interfaces/ISablierV2MerkleLockupFactory.sol

```diff
@@ -37,6 +37,18 @@ interface ISablierV2MerkleLockupFactory {
         uint256 recipientCount
     );
 
+    /*//////////////////////////////////////////////////////////////////////////
+                                 CONSTANT FUNCTIONS
+    //////////////////////////////////////////////////////////////////////////*/
+
+    /// @notice Verify the sum of percentages in `tranches` equals 100% , i.e. 1e18.
+    /// @dev Reverts if the sum of percentages overflows.
+    /// @param tranches The tranches with their respective unlock percentages.
```

Comment:
missing `@return` tag

Reply:
cc @andreivladbrg https://github.com/sablier-labs/v2-periphery/pull/357#discussion_r1647794963

Reply:
I'd like to keep the return tag because it's not clear from the `@notice` what the verification process entails